src/store/middlewares/peers.js

```diff
@@ -7,8 +7,23 @@ import { shouldAutoLogIn, getAutoLogInData } from './../../utils/login';
 const peersMiddleware = store => next => (action) => {
   next(action);
 
-  const network = Object.assign({}, getNetwork(networks.mainnet.code));
   const autologinData = getAutoLogInData();
+  const { loginUrl } = autologinData;
+
+  let loginNetwork = Object.entries(networks).find((network) => {
+    const { nodes } = network.slice(-1).shift();
+    return Array.isArray(nodes) ? nodes.includes(loginUrl) : false;
+  });
```

Comment:
These lines are duplicated also in here
https://github.com/LiskHQ/lisk-hub/blob/544d84ce317ff6aa3bc2b270766fd56c37e4270d/src/components/login/login.js#L29-L34

Looks like something that could have a util function in `src/utils/login`